fix-point/src/CopyFiles.hs

```diff
@@ -0,0 +1,45 @@
+module CopyFiles where
+
+import Control.Applicative ((<$>))
+import Control.Exception (throw)
+import Control.Monad (when,forM_)
+import Data.Functor ((<$>))
+import System.Directory
+import System.FilePath ((</>))
+-- import System.File.Tree (getDirectory, copyTo_)
+
+import qualified Rename as R
+
+-- getDirectory :: FilePath -> IO System.File.Tree.FSTree
+-- copyTo_ :: FilePath -> System.File.Tree.FSTree -> IO ()
+-- copyDir :: FilePath -> FilePath -> IO ()
+-- copyDir source target = getDirectory source >>= copyTo_ target
+
+copyAll :: FilePath -> FilePath -> IO ()
+copyAll source target = do
+  -- when :: Applicative f => Bool -> f () -> f ()
```

Comment:
This comment with type signature looks redundant. It's okay to have them during your intermediate work but projects usually should be polished before release (in your case before commit)